Tests/SafeDIMacrosTests/InstantiableMacroTests.swift

```diff
@@ -777,14 +777,46 @@ final class InstantiableMacroTests: XCTestCase {
         assertMacro {
             """
             @Instantiable
-            public enum ExampleService {}
+            public enum ExampleService: Instantiable {}
```

Comment:
added these `Instantiable` conformances while I was here. It helps make this test more resilient to underlying code ordering.